main/src/TableSchemaValidator.json

```diff
@@ -0,0 +1,213 @@
+        },
+        "minimum-constraint": {
+          "name": "Minimum Constraint",
+          "type": "schema",
+          "context": "body",
+          "weight": 7,
+          "message": "The value {value} in row {row_number} and column {column_number} does not conform to the minimum constraint of {constraint}",
+          "description": "This field value should be greater or equal than constraint value.\n\n How it could be resolved:\n - If this value is not correct, update the value.\n - If value is correct, then remove or refine the `minimum` constraint in the schema.\n - If this error should be ignored disable `minimum-constraint` check in {validator}."
+        },
+        "maximum-constraint": {
+          "name": "Maximum Constraint",
+          "type": "schema",
+          "context": "body",
+          "weight": 7,
+          "message": "The value {value} in row {row_number} and column {column_number} does not conform to the maximum constraint of {constraint}",
+          "description": "This field value should be less or equal than constraint value.\n\n How it could be resolved:\n - If this value is not correct, update the value.\n - If value is correct, then remove or refine the `maximum` constraint in the schema.\n - If this error should be ignored disable `maximum-constraint` check in {validator}."
+        },
+        "minimum-length-constraint": {
+          "name": "Minimum Length Constraint",
+          "type": "schema",
+          "context": "body",
+          "weight": 7,
+          "message": "The value {value} in row {row_number} and column {column_number} does not conform to the minimum length constraint of {constraint}",
+          "description": "A lenght of this field value should be greater or equal than schema constraint value.\n\n How it could be resolved:\n - If this value is not correct, update the value.\n - If value is correct, then remove or refine the `minimumLength` constraint in the schema.\n - If this error should be ignored disable `minimum-length-constraint` check in {validator}."
+        },
+        "maximum-length-constraint": {
+          "name": "Maximum Length Constraint",
+          "type": "schema",
+          "context": "body",
+          "weight": 7,
+          "message": "The value {value} in row {row_number} and column {column_number} does not conform to the maximum length constraint of {constraint}",
+          "description": "A lenght of this field value should be less or equal than schema constraint value.\n\n How it could be resolved:\n - If this value is not correct, update the value.\n - If value is correct, then remove or refine the `maximumLength` constraint in the schema.\n - If this error should be ignored disable `maximum-length-constraint` check in {validator}."
+        }
+
+
+    }
+}
```

Comment:
I don't think this file is in an appropriate location - this folder is for Java source files.

Reply:
moved

Reply:
thanks. Any idea how we could localize this? Could this be done UI-side?

Reply:
No. this is server-side resource. we don't have the server-side i18n support yet. Can be added as a infrastructure and then do the localization.

Reply:
or it should also be possible to use this file directly from the UI… but okay, let's keep this for now.